### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by TeddyTesfaye**
#1473, #1475, #1476 
@tjwdev 
@tsegay 
@kasimhelil 
@originalname51

### CODE REVIEW

File: gtas-parent/checkstyle.xml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<!DOCTYPE module PUBLIC
```
Comment (originalname51) [inline gtas-parent/checkstyle.xml]: Was this causing issues? It does not appear to be related to other code but is deleted.

Comment (TeddyTesfaye) [reply gtas-parent/checkstyle.xml]: No it did not. Hmm.. Let me see why it is deleted
